scripts/src/java/org/oppia/android/scripts/proto/todo_exemptions.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+// Exemptions for the TODO check.
+message TodoExemptions {
```

Comment:
Similar feedback to the other PR, but I think we also will want to make sure the check ensures the exemption file is correct (so if lines move around previous TODO exemptions should fail), and that there are tests verifying that the check class does this as expected.

Reply:
Done.